Herbaria-Project-Outline.md

```diff
@@ -0,0 +1,97 @@
+# Herbaria Project Outline
+
+### *Smriti Suresh, Dima Kazlouski, Douglas Moy - 2023-10-06 v1.0.0-dev*
+
+## Overview
+
+The changing climate increases stressors that weaken plant resilience, disrupting forest structure and ecosystem services. Rising temperatures lead to more frequent droughts, wildfires, and invasive pest outbreaks, leading to the loss of plant species. That has numerous detrimental effects, including lowered productivity, the spread of invasive plants, vulnerability to pests, altered ecosystem structure, etc. The project aims to aid climate scientists in capturing patterns in plant life concerning changing climate.
+The herbarium specimens are pressed plant samples stored on paper. The specimen labels are handwritten and date back to the early 1900s. The labels contain the curator's name, their institution, the species and genus, and the date the specimen was collected. Since the labels are handwritten, they are not readily accessible from an analytical standpoint. The data, at this time, cannot be analyzed to study the impact of climate on plant life.
+The digitized samples are an invaluable source of information for climate change scientists, and are providing key insights into biodiversity change over the last century. Digitized specimens will facilitate easier dissemination of information and allow more people access to data. The project, if successful, would enable users from various domains in environmental science to further studies pertaining to climate change and its effects on flora and even fauna.
+
+
+### A. Problem Statement:
+
+Develop a machine learning solution for automating the extraction and digitization of critical information from handwritten labels on historical herbarium specimens, dating back to the early 1900s. Other potential problems to tackle are segmeneting non-relevant parts of the image, and building a classifier for the image itself. 
```

Comment:
The goal is extract both handwritten and typed text, not just the handwritten. As discussed there are really two goals:

1. Improve recognition results on handwritten text, since that is not as accurate as for typed text.
2. Extract additional information in addition to the plant species.

Also, segmenting the image is not a goal for the project this term.